CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log for rocm-cmake
 
+## [0.6.1]
+### Fixed
+- Add `cmake_minimum_required` check to `ROCMInstallTargets.cmake` to fix error with `CMP0057` not set
+- Unset `CPACK_PACKAGING_INSTALL_PREFIX` when `CPACK_USE_DESTDIR` is set to fix error with TGZ generator
```

Comment:
Not sure this needs to count as a separate version, there has been no release 0.6.0